### DISCUSSION TITLE
Daikin Altherma 3 instellen in Home Assistant

### DISCUSSION THREAD

**Discussion by JanB19** [community/community#123164] [Other Feature Feedback, Questions, & Ideas]
Thank you have removed it

**Comment by LiteBrite82** [comment]
Hi @JanB19 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the manufacturers support page. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. I have moved this discussion over to `General` for greater visibility.

Best of luck!
